haiku-apps/minimizeall/minimizeall-1.0.0.recipe

```diff
@@ -65,4 +68,4 @@ INSTALL()
 	mkdir -p "$dataDir/deskbar/menu/Desktop applets"
 	symlinkRelative -s "$appsDir/MinimizeAll/DeskbarAddOn"  \
 		"$dataDir/deskbar/menu/Desktop applets/MinimizeAll"
-}
+}
```

Comment:
The last line of the recipe is missing a linefeed.
You can fix it by editing the file with vi and saving it.